markdown_it/renderer.py

```diff
@@ -235,16 +235,11 @@ def fence(self, tokens, idx, options, env):
         # May be, one day we will add .clone() for token and simplify this part, but
         # now we prefer to keep things local.
         if info:
-            i = token.attrIndex("class")
             tmpAttrs = token.attrs[:] if token.attrs else []
 
-            if i < 0:
-                tmpAttrs.append(["class", options.langPrefix + langName])
-            else:
-                tmpAttrs[i][1] += " " + options.langPrefix + langName
```

Comment:
This line actually seems to have a bug that we're fixing by using the immutable tuple :smile: Mutating `tmpAttrs[i]` here will also mutate `token.attrs[i]` because `tmpAttrs` is a copy, not a deep copy, of `token.attrs`.

Reply:
Here's the fix in JS markdown-it https://github.com/markdown-it/markdown-it/commit/c9949dd585315990689a6bb361388ea30c94638a

If we don't end up merging this PR, we should do the same change.

Reply:
This fix is now in merged in `master` of markdown-it-py btw